src/app/reviews/reviews.component.html

```diff
@@ -0,0 +1,23 @@
+<ion-header>
+  <ion-toolbar>
+    <ion-title center>Reviews</ion-title>
+  </ion-toolbar>
+</ion-header>
+
+<ion-content padding color="light">
+  <div class="top-round-btns" text-center>
+    <ion-button class="do" [ngClass]="{'active-review': activeToggle}" (click)="toggleActiveDo()">Todo</ion-button>
+    <ion-button class="done" [ngClass]="{'active-review': !activeToggle}" (click)="toggleActiveDone()">Done</ion-button>
+  </div>
+  <div *ngIf="!reviews.length; else reviewsCard" class="no-review-show" padding text-center>
+    <div class="empty-status" >
+      <h2 text-center>You have no new submissions to review</h2>
+    </div>
+  </div>
+  <ng-template #reviewsCard>
+    <ng-container *ngFor="let review of reviews">
+       <app-activity-card [review]="review" [toggle]="activeToggleDo"></app-activity-card>
```

Comment:
General suggestion (not only for this line):
The logic you used here is to pass `review` and `toggle` to `activity-card` and check whether to show this card or not in ActivityCardComponent. 
Since ActivityCardComponent is a separate component, what this component do should be just getting the necessary information and display it. "Whether show this card or not" this logic belongs to the review page only, so it should be placed here, not as part of the activity card component.
ActivityCardComponent should only handle the layout.